src/mongo/db/backup/backup_commands.cpp

```diff
@@ -43,12 +44,13 @@ class CreateBackupCommand : public Command {
              << std::endl
              << "{ createBackup: 1, backupDir: <destination directory> }";
     }
-    void addRequiredPrivileges(const std::string& dbname,
-                               const BSONObj& cmdObj,
-                               std::vector<Privilege>* out) override {
-        ActionSet actions;
-        actions.addAction(ActionType::startBackup);
-        out->push_back(Privilege(parseResourcePattern(dbname, cmdObj), actions));
+    Status checkAuthForCommand(ClientBasic* client,
+                               const std::string& dbname,
+                               const BSONObj& cmdObj) override {
+        return AuthorizationSession::get(client)->isAuthorizedForActionsOnResource(
+                   ResourcePattern::forAnyNormalResource(), ActionType::startBackup)
+            ? Status::OK()
+            : Status(ErrorCodes::Unauthorized, "Unauthorized");
```

Comment:
No, it wasn't. But the changes are done because:
- it's the new recommended way of auth check (`addRequiredPrivileges` is obsolete)
- the privilege was previously checked against only one database (admin), not against all databases (which is `forAnyNormalResource`)